### DISCUSSION TITLE
Does github package support sbt plugin?

### DISCUSSION THREAD

**Discussion by chenrui333** [community/community#25679] [Actions]
<p>:wave: I am trying to migrate my sbt plugins off Bintray (as it goes away). To my best knowledge, I can only migrate those plugin to be hosted on Sonatype rather than using Github Package registry.</p>
<p>I want to double confirm in here. Thanks a lot!</p>

**Comment by mickeygousset** [comment]
<p>So I don’t know about sbt, but GitHub Packages supports the following:</p>
<p>Docker<br>
RubyGems<br>
npm<br>
Apache Maven<br>
Gradle<br>
Nuget</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/packages/learn-github-packages/about-github-packages" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/packages/learn-github-packages/about-github-packages" target="_blank" rel="noopener">About GitHub Packages - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/packages/learn-github-packages/about-github-packages" target="_blank" rel="noopener">//docs.github.com/en/packages/learn-github-packages/about-github-packages</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by laughedelic** [comment, marked as answer]
<p>There is an sbt plugin for publishing to GitHub Packages:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/djspiewak/sbt-github-packages" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/766656/181092473-99aeb771-0bda-4dd3-8dd3-cfe387dffa49.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/a/6a87592aa8989952295ce517b07a48c76b43efbc_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/a/6a87592aa8989952295ce517b07a48c76b43efbc_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/a/6a87592aa8989952295ce517b07a48c76b43efbc.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/a/6a87592aa8989952295ce517b07a48c76b43efbc_2_10x10.png"></div>

<h3><a href="https://github.com/djspiewak/sbt-github-packages" target="_blank" rel="noopener nofollow ugc">djspiewak/sbt-github-packages</a></h3>


  <p><span class="label1">A simple sbt plugin for publishing to GitHub Packages, in the style of sbt-sonatype and sbt-bintray</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by chenrui333** [comment]
<p>Scala version would be our issue of using this plugin. But it is a good plugin though.</p>

**Comment by laughedelic** [comment]
<p>Scala version you use for your project and the Scala version used by sbt and its plugins are independent of each other. So it doesn’t matter which Scala version you are using, you should be able to add this plugin to your build.</p>
